plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -521,8 +521,10 @@ function MosaicMenuItem:update()
             },
         }
     else
-        if file_mode ~= "file" then
-            self.file_deleted = true
+        local filemanager = require("apps/filemanager/filemanager").instance
+        local file_selected = filemanager.selected_files and filemanager.selected_files[self.filepath]
+        if file_mode ~= "file" or file_selected then
+            self.file_deleted = true -- dim file
```

Comment:
I believe (quickly looking at the code) that in this MosaicMenuItem object, the FileChooser instance is available as `self.menu`.
So, may be you can reach `selected_files` with just `self.menu.filemanager.selected_files` (instead of having to require().instance in each MosaicMenuItem).

Reply:
Yeah, thanks!